discord/audit_logs.py

```diff
@@ -318,7 +318,7 @@ def __init__(
                 "$remove_allow_list",
             ]:
                 self._handle_trigger_metadata(
-                    self.after, self.before, entry, elem["new_value"], attr
+                    self.after, self.before, entry, elem["new_value"], attr  # type: ignore
```

Comment:
What's being ignored here? (And the one above)

Reply:
elem["new_value"], kind of logic from lines above as we get warning about type here also:
![image](https://github.com/Pycord-Development/pycord/assets/61660055/ee503ee2-5560-44ce-8a9e-31f2aa055373)